2022-01-18 11:27:44,774 INFO    MainThread:166490 [wandb_setup.py:_flush():71] setting env: {}
2022-01-18 11:27:44,774 INFO    MainThread:166490 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-18 11:27:44,774 INFO    MainThread:166490 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-18 11:27:44,774 INFO    MainThread:166490 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220118_112744-2zi09l9h/logs/debug.log
2022-01-18 11:27:44,774 INFO    MainThread:166490 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220118_112744-2zi09l9h/logs/debug-internal.log
2022-01-18 11:27:44,775 INFO    MainThread:166490 [wandb_init.py:init():404] calling init triggers
2022-01-18 11:27:44,775 INFO    MainThread:166490 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-18 11:27:44,775 INFO    MainThread:166490 [wandb_init.py:init():460] starting backend
2022-01-18 11:27:44,775 INFO    MainThread:166490 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-18 11:27:44,788 INFO    MainThread:166490 [backend.py:ensure_launched():216] starting backend process...
2022-01-18 11:27:44,793 INFO    MainThread:166490 [backend.py:ensure_launched():221] started backend process with pid: 166545
2022-01-18 11:27:44,793 INFO    MainThread:166490 [wandb_init.py:init():469] backend started and connected
2022-01-18 11:27:44,795 INFO    MainThread:166490 [wandb_init.py:init():533] updated telemetry
2022-01-18 11:27:44,803 INFO    MainThread:166490 [wandb_init.py:init():563] communicating current version
2022-01-18 11:27:45,172 INFO    MainThread:166490 [wandb_init.py:init():568] got version response 
2022-01-18 11:27:45,172 INFO    MainThread:166490 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-18 11:27:45,347 INFO    MainThread:166490 [wandb_init.py:init():606] starting run threads in backend
2022-01-18 11:27:45,420 INFO    MainThread:166490 [wandb_run.py:_console_start():1810] atexit reg
2022-01-18 11:27:45,420 INFO    MainThread:166490 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-18 11:27:45,420 INFO    MainThread:166490 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-18 11:27:45,421 INFO    MainThread:166490 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-18 11:27:45,421 INFO    MainThread:166490 [wandb_init.py:init():633] run started, returning control to user process
2022-01-18 11:27:46,864 INFO    MainThread:166490 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-18 11:27:46,864 INFO    MainThread:166490 [wandb_run.py:_restore():1752] restore
2022-01-18 11:27:49,037 INFO    MainThread:166490 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 777
  total_bytes: 777
}

2022-01-18 11:27:49,218 INFO    MainThread:166490 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 777
  total_bytes: 777
}

2022-01-18 11:27:49,363 INFO    MainThread:166490 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 777
  total_bytes: 2139
}

2022-01-18 11:27:49,466 INFO    MainThread:166490 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 777
  total_bytes: 7884
}

2022-01-18 11:27:49,569 INFO    MainThread:166490 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7884
  total_bytes: 7884
}

2022-01-18 11:27:49,673 INFO    MainThread:166490 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7884
  total_bytes: 7884
}

2022-01-18 11:27:49,776 INFO    MainThread:166490 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7884
  total_bytes: 7884
}

2022-01-18 11:27:49,880 INFO    MainThread:166490 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7884
  total_bytes: 7884
}

2022-01-18 11:27:49,983 INFO    MainThread:166490 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7884
  total_bytes: 7884
}

2022-01-18 11:27:50,259 INFO    MainThread:166490 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7884
  total_bytes: 7884
}

2022-01-18 11:27:50,628 INFO    MainThread:166490 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7884
  total_bytes: 7884
}
local_info {
}

2022-01-18 11:27:51,677 INFO    MainThread:166490 [wandb_run.py:_append_files():2180] logging synced files
